source/features/extend-diff-expander.css

```diff
@@ -19,3 +19,26 @@
 	);
 	cursor: pointer;
 }
+
+/* Support new Commit view */
+.rgh-extend-diff-expander .diff-line-row:has(button[data-direction]):hover {
+	color: var(--diffBlob-hunkNum-fgColor-hover, var(--fgColor-onEmphasis, #fff));
+	background: var(
+		--diffBlob-hunkNum-bgColor-hover,
+		var(--bgColor-accent-emphasis, #0366d6)
+	);
+	cursor: pointer;
+	transition: var(--duration-fast) var(--easing-easeInOut);
+	transition-property: color, fill, background-color, border-color;
+
+	& button[data-direction] {
+		color: var(
+			--diffBlob-hunkNum-fgColor-hover,
+			var(--fgColor-onEmphasis, #fff)
+		);
+		background: var(
+			--diffBlob-hunkNum-bgColor-hover,
+			var(--bgColor-accent-emphasis, #0366d6)
```

Comment:
Why this var fallback? Was --diffBlob missing somewhere?

Also don't forget to use `fuchsia` as last fallback everywhere.

Reply:
<img width="431" alt="image" src="https://github.com/user-attachments/assets/ca82cba7-e6fe-436b-a870-faa6066d4a0a">